prover/src/trace/trace_lde/default/tests.rs

```diff
@@ -31,6 +31,7 @@ fn extend_trace_table() {
         trace.info(),
         trace.main_segment(),
         &domain,
+        1,
```

Comment:
Should this not be `trace.info().main_trace_width()`?

Reply:
Indeed, thank you